style.css

```diff
@@ -356,27 +362,99 @@
   font-size: 30px;
 }
 
-.recent-work button:hover {
-  color: #ff6b00;
-  border: 1px solid #f55800;
-  background: #fff5e1;
+footer {
+  width: 100%;
+  height: 100vh;
+  padding: 120px 24px;
 }
 
-.work-container > div button:hover {
-  color: #ff6b00;
-  border: 1px solid #f55800;
-  background: #fff5e1;
+.icons {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  gap: 1.5rem;
+}
+
+footer div.icons > ul {
+  display: flex;
+  justify-content: space-around;
+  width: 50%;
+}
+
+footer div.icons > ul > li {
+  list-style-type: none;
+  padding: 1rem 0;
+}
+
+footer div.icons h2.attract {
+  font-family: 'Inter', sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  margin-bottom: 1rem;
+}
```

Comment:
- Kindly make your test in bold as it required by the previous reviewer so that it will make your design matches the Figma design :+1: :grinning: 
 ## your design 
![image](https://user-images.githubusercontent.com/95297251/186550343-08b75eea-0119-40b9-8248-e838fc655146.png)


## Figma design

![image](https://user-images.githubusercontent.com/95297251/186550421-8726a07d-d8f3-42c2-928c-6b3556526b3a.png)